pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ConsumerConfigurationData.java

```diff
@@ -134,6 +131,7 @@ public int getMaxPendingChuckedMessage() {
 
     private RegexSubscriptionMode regexSubscriptionMode = RegexSubscriptionMode.PersistentOnly;
 
+    @JsonIgnore
```

Comment:
Will this introduce breaking change?

Reply:
I think it's not needed anymore since https://github.com/apache/pulsar/issues/16487